components/counter/readme.md

```diff
@@ -0,0 +1,40 @@
+# Counter
+
+The `Counter` component let's you use circular progress bar with two props - `count` and `limit`.
+
+This also exposes another component named `CircularProgressBar`, which can be used when we don't want the text annotation and just the SVG illustration.
+
+## Usage
+
+```js
+import { Counter, CircularProgressBar } from '@10up/block-components';
+
+function MyComponent( props ) {
+
+    return (
+		<Counter
+			count={text.length}
+			limit={20}
+		/>
+
+		<CircularProgressBar
+			percentage={(text.length / 20) * 100}
+		/>
```

Comment:
[nit] Needs more indentation here

Reply:
I missed wrapping it by Fragment! 🙈 
Fixed it now. 👍 Hopefully indentation is also fine now.